utils/generate_types/overrides-test.d.ts

```diff
@@ -60,6 +60,7 @@ type LiteralUnion<T extends U, U = string> = T | (U & { zz_IGNORE_ME?: never });
 interface TestConfig {
   reporter?: LiteralUnion<'list'|'dot'|'line'|'github'|'json'|'junit'|'null'|'html', string> | ReporterDescription[];
   webServer?: TestConfigWebServer | TestConfigWebServer[];
+  workers?: number | string;
```

Comment:
I think you don't need to add this line, types generator should pick it up automatically.

Reply:
Done.